### PR TITLE
Feature/customize button

### PR DESCRIPTION
**Pull request by blaurenzano**
I've added customization options for the chat button, including language, texts for the button, and color of the button.
It can be tested in this workspace: https://chat--babycottonsar.myvtex.com/
image
image

This would be a patch release.

### CODE REVIEW

File: pixel/head.html [hunk 1]
```diff
@@ -3,14 +3,44 @@
...
+				}
+				.embeddedServiceHelpButton .helpButton .uiButton:focus {
+					outline: 1px solid ${color} !important;
+				}
+			`;
+
+			const styleSheet = document.createElement("style");
+			styleSheet.type = "text/css";
+			styleSheet.innerText = styles;
+			document.head.appendChild(styleSheet);
```
Comment (klzns) [inline pixel/head.html]: That's weird.. Doesnt the settings have a theme settings?

**Review by klzns — CHANGES_REQUESTED** [review]

File: pixel/head.html (hunk 1 above)
Comment (klzns) [reply pixel/head.html]: Also, you shouldn't release a new version in a branch hehe

only the bot should do it

Comment (blaurenzano) [reply pixel/head.html]: Thanks Breno, I agree it's weird adding styles that way. I've just removed it, we'll see if that can be done in other ways later.
I didn't understand what you said about the new version release, I didn't change the version number in the manifest, it's still 0.0.3, not sure what you meant.

Comment (klzns) [reply pixel/head.html]: I mean this commit here:
https://github.com/vtex-apps/salesforce-chat/pull/5/commits/eaa13631b5bc1d3df6b69135a55c536d55338943

File: pixel/body.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        '{{settings.enabledFeatures}}'
+      )
+      embedded_svc.settings.enabledFeatures = JSON.parse(enabledFeatures)
+      embedded_svc.settings.entryFeature = '{{settings.entryFeature}}'
+
+      embedded_svc.init(
+        window.decodeURIComponent('{{settings.urlOne}}'),
+        window.decodeURIComponent('{{settings.urlTwo}}'),
+        gslbBaseURL,
+        '00DP0000000Dhn3',
```
Comment (klzns) [inline pixel/body.html]: What is this value? Is that specific to this store?

**Review by klzns — CHANGES_REQUESTED** [review]
Hi, thanks! I pushed forced your branch to commit some changes.

To continue developing on it, please run:
```
git fetch
git reset origin/feature/customize-button --hard
```

Left a comment to review.

File: pixel/head.html (hunk 1 above)
Comment (blaurenzano) [reply pixel/head.html]: Oh that was @ccamargo2294 in the previous release.

File: pixel/body.html (hunk 2 above)
Comment (blaurenzano) [reply pixel/body.html]: Not sure, but we could make it customizable too.

Comment (klzns) [reply pixel/body.html]: It sounds like an ID. Maybe it should be configurable.

Comment (blaurenzano) [reply pixel/body.html]: Allright, I'm on it.

Comment (blaurenzano) [reply pixel/body.html]: Done!

**Review by klzns — APPROVED** [review]

File: manifest.json [hunk 3]
```diff
@@ -113,13 +113,17 @@
...
       },
-      "urlOne": {
+      "baseCoreURL": {
         "type": "string",
-        "title": "Url 1"
+        "title": "baseCoreURL"
       },
-      "urlTwo": {
+      "communityEndpointURL": {
         "type": "string",
```
Comment (klzns) [inline manifest.json:121]: Ops, that's a breaking change. 

Do you want to keep this change and deploy a major version of the app?

Or do you want to use the previous name and keep the same major version?

**Review by klzns — CHANGES_REQUESTED** [review]

File: manifest.json (hunk 3 above)
Comment (klzns) [reply manifest.json:121]: A new major version of the app will require stores that use this app to install the app again to use the new version.

Keeping the major version, the stores that use this app will get it automatically.

Comment (blaurenzano) [reply manifest.json:121]: If it's just reinstalling the app I think it's ok. Let's go with the major version change.

**Review by klzns — APPROVED** [review]

**Comment by klzns** [comment]
@allcontributors add @blaurenzano for code

**Comment by blaurenzano** [comment]
Thanks!

**Pull request merged**